server/src/core/entry_point.rs

```diff
@@ -148,8 +148,10 @@ impl EntryPointMgr {
         EntryPointMgr::create_dir_symbols_from_path_to_entry(session, &PathBuf::from(path), entry)
     }
 
-    pub fn create_new_custom_entry_for_path(session: &mut SessionInfo, path: &String) -> bool {
-        let path_sanitized = PathBuf::from(path).sanitize();
+    /// Create a new custom entry point for a given tree path and file path.
+    /// tree_path can possibly be the path stripped from __manifest__/__init__.py
+    pub fn create_new_custom_entry_for_path(session: &mut SessionInfo, tree_path: &String, file_path: &String) -> bool {
+        let path_sanitized = PathBuf::from(tree_path).sanitize();
```

Comment:
Why do you sanitize it again if all call to the function sanitize it already? Usually each function assume that path is sanitized or not

Reply:
it was already there, but it is indeed unnecessary